src/core/thread/mle_router.cpp

```diff
@@ -184,6 +184,24 @@ Error MleRouter::SetRouterEligible(bool aEligible)
     return error;
 }
 
+// If the router was a leader or had more than 5 children prior to reset,
+// the multicast link request is retransmitted as a critical message
+void MleRouter::SetLinkRequestTransmissionCounter(void)
+{
+    Settings::NetworkInfo networkInfo;
+    uint16_t              numOfChildren = mChildTable.GetNumChildren(Child::kInStateValidOrRestoring);
+    mLinkRequestAttempts                = kMaxTransmissionCount;
```

Comment:
newline after var declaration?

```suggestion
    uint16_t              numOfChildren = mChildTable.GetNumChildren(Child::kInStateValidOrRestoring);
    
    mLinkRequestAttempts = kMaxTransmissionCount;
```